var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -462,6 +463,10 @@ def configure_args(self):
         if 'schedulers=auto' not in spec:
             config_args.extend(self.with_or_without('schedulers'))
 
+        if '~gpfs' in spec:
+            os.environ['ac_cv_header_gpfs_h'] = 'no'
+            os.environ['ac_cv_header_gpfs_fcntl_h'] = 'no'
```

Comment:
It might be better to move this to `setup_build_environment` so we can modify Spack's env object.

Reply:
@adamjstewart I'm new to `setup_build_environment`; so let me know if this is how it should be done.